onnxruntime/core/providers/webgpu/tensor/flatten.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/framework/op_kernel.h"
+#include "core/providers/cpu/nn/flatten.h"
+#include "core/framework/data_transfer_manager.h"
+
+namespace onnxruntime {
+namespace webgpu {
+
+class Flatten final : public OpKernel {
+ public:
+  explicit Flatten(const OpKernelInfo& info) : OpKernel{info} {
+    ORT_ENFORCE(info.GetAttr<int64_t>("axis", &axis_).IsOK());
```

Comment:
axis attribute is optional with default value 1. In this case we want to use GetAttrDefault instead of GetAttr, which fails if the attribute is not specified.

Reply:
When I try to use GetAttrDefault I get the below error:
![{5DEC7C46-15BE-4EAB-81D2-E7DB2B4FD83E}](https://github.com/user-attachments/assets/8e97649f-6df3-4118-8312-4be0f1f7ade2)

Additionally, there is [an existing test](https://github.com/microsoft/onnxruntime/blob/e8bf46a70ea532af0e4850ee31de7ad21b92d6c4/onnxruntime/test/providers/cpu/nn/flatten_op_test.cc#L34) for default axis and it passes while using GetAttr.

Reply:
I think it is `GetAttrOrDefault`